Template/Tag/ShareaholicTag.web.js

```diff
@@ -0,0 +1,41 @@
+(function () {
+    return function (parameters, TagManager) {
+        this.fire = function () {
+            var siteId = parameters.get('shareaholicSiteId');
+            if (siteId) {
+                // should result in a <script> like this:
+                // <script type="text/javascript" data-cfasync="false" src="//apps.shareaholic.com/assets/pub/shareaholic.js" data-shr-siteid="[SITEID]" async="async"></script>
+
+                var script = document.createElement('script');
+                var s = document.getElementsByTagName('script')[0];
```

Comment:
do we need to fallback to `document.body` or `document.head` if no script element can be found?

Reply:
The Matomo tracking code does it the same (but is also a `<script>` itself). But I guess it is okay, as using MTM without any script tag sounds unlikely (let's hope no one uses only a `onload=""`)

Reply:
👍